README.md

```diff
@@ -535,6 +533,8 @@ For information about the governance of the Node.js project, see
   **Robert Jefe Lindstaedt** <<robert.lindstaedt@gmail.com>>
 * [estliberitas](https://github.com/estliberitas) -
   **Alexander Makarenko** <<estliberitas@gmail.com>>
+* [Ethan-Arrowood](https://github.com/Ethan-Arrowood) -
+  **Ethan Arrowood** <<ethan@arrowood.dev>> (he/him)
```

Comment:
Well there’s clearly a bug in this script. It needs to check that someone _has been_ a collaborator for at least the length of time that it checks for inactivity.

Reply:
The problem is that this email address hasn't been used recently. It should be fixed by an update to https://github.com/nodejs/node/blob/main/.mailmap

Reply:
@Ethan-Arrowood can you please open a PR to map the different email addresses you use to commit to the one you use in the collaborator list please?

Reply:
fwiw I know Ethan is on vacation right now and might not be paying attention to GitHub pings 😄

Reply:
Ack! Online tomorrow and will get this fixed up asap. Will also make sure to stick to my personal email for all future commits